frontend/src/views/Game.tsx

```diff
@@ -180,26 +192,35 @@ function GamePage() {
 
     // Get the result of promises and set the default code list.
     Promise.all(promises).then((result) => {
-      const codeList = [];
+      const newCodeList = [];
+      const newLanguageList = [];
+      const codeMap = result;
 
       for (let i = 0; i < result.length; i += 1) {
-        codeList.push(result[i][language[i]]);
+        newCodeList.push(result[i][Language.Python]);
+        newLanguageList.push(Language.Python);
       }
```

Comment:
What is happening here - why is only `Language.Python` being used?

Reply:
This creating the starting code list and language list for the frontend page. So this essentially just grabs default code from the Python code for each problem from the backend request, adds it to the code list, and adds python to the language list. I'll switch this to Java.